src/libserver/cfg_utils.c

```diff
@@ -816,16 +816,6 @@ rspamd_config_post_load (struct rspamd_config *cfg,
 	rspamd_regexp_library_init (cfg);
 	rspamd_multipattern_library_init (cfg->hs_cache_dir);
 
-#ifdef WITH_HYPERSCAN
-	if (!cfg->disable_hyperscan) {
-		if (!(cfg->libs_ctx->crypto_ctx->cpu_config & CPUID_SSSE3)) {
-			msg_warn_config ("CPU doesn't have SSSE3 instructions set "
-					"required for hyperscan, disable it");
-			cfg->disable_hyperscan = TRUE;
-		}
-	}
-#endif
```

Comment:
This should not be removed - it is an used feature to disable HS from the configuration. SSSE3 should also be checked for x86/x86_64 architectures.